src/modules/infobox/components/FeatureProfile.vue

```diff
@@ -53,11 +53,6 @@
                 data-cy="profile-popup-csv-download-button"
                 @click="onCSVDownload"
             />
-            <ButtonWithIcon
-                :button-font-awesome-icon="['far', 'trash-alt']"
-                data-cy="profile-popup-delete-button"
-                @click="onDelete"
-            />
```

Comment:
I think this should stay. How are we supposed to remove a measure line without this button?

Reply:
@pakb See screenshot below, this line remove the button crossed in red, but you still have the button in the description popup to delete the line. I don't think we need 2 buttons here.
![Uploading profile-delete-button.png…]()

Reply:
that works for a line yes, and it makes sense to remove the duplicate in this case, but what about measure?
![image](https://user-images.githubusercontent.com/11540521/199476485-585b4ffb-b82d-4504-b882-a1ad9dca72ba.png)

Reply:
Very good point ! I didn't know about this use case. So I'll check if I can easily remove only for the line otherwise I leave it.